site/public/css/email-status.css

```diff
@@ -66,9 +66,8 @@
 }
 
 .page-btn {
-    background-color: var(--text-black);
-    color: var(--focus-actionable-blue);
-    border: 1px solid var(--text-black);
```

Comment:
is this a typo that the border color specification has been deleted?

Reply:
In my wave testing I found that WAVE threw contrast errors on navigation buttons for this page while in dark mode, and the buttons had solid black background in light mode, which I felt looked odd. They look like this now:
![image](https://github.com/Submitty/Submitty/assets/89590361/34842e96-daed-454d-b93f-59f6b5243710)
![image](https://github.com/Submitty/Submitty/assets/89590361/f6462dff-4fe4-4862-8c91-232fc022cec4)
![image](https://github.com/Submitty/Submitty/assets/89590361/250c0af9-775a-4a10-8db4-0c71d70ff369)
The borders don't appear in dark mode, but I thought it still looked good and it resolved the contrast errors. I can change it if needed.

Reply:
In general, making small bug fixes in large refactor PRs like this can lead to bugs which are difficult to track down.  In the future, it might be better to perform the refactor in one PR, and then address the WAVE issue in a separate PR so if the bug fix needs to be reverted, we don't have to revert the entire refactor.